descriptors/ModuleDescriptor-template.json

```diff
@@ -539,7 +542,8 @@
           "modulePermissions": [
             "inventory-storage.holdings.item.put",
             "inventory-storage.holdings.collection.get",
-            "inventory-storage.instances.item.get"
+            "inventory-storage.instances.item.get",
+            "user-tenants.collection.get"
```

Comment:
this permission is used by mod-orders-storage Holding Create consumer in ECS mode:
https://github.com/folio-org/mod-orders-storage/blob/c0923e3a120d7108241e0ecc1f7a22cb7da246ae/src/main/java/org/folio/event/handler/HoldingCreateAsyncRecordHandler.java#L50